.gitignore

```diff
@@ -10,10 +10,11 @@ packages/*/coverage
 
 # production
 /build
-packages/*/lib
+packages/**/lib
```

Comment:
There are a few other `packages/*/folder`s that I assume need to be updated as well...

- `packages/*/coverage`
- `packages/*/.rollup.cache`